ctru-rs/src/lib.rs

```diff
@@ -26,6 +26,12 @@ pub fn init() {
         libc::atexit(services_deinit);
     }
 
+    #[cfg(not(test))]
+    _panic_hook_setup();
+}
+
+fn _panic_hook_setup() {
```

Comment:
nit: This shouldn't be prefixed with `_`. It's already a private function because it doesn't have `pub`.